java/test/eda/sorting/SortingTest.java

```diff
@@ -0,0 +1,86 @@
+package eda.sorting;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import java.util.Arrays;
+import java.util.Collection;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+
+@RunWith(value = Parameterized.class)
+public class SortingTest {
+
+    private Sorting sortingTest;
+
+    public SortingTest(Sorting sortingTest) {
+        this.sortingTest = sortingTest;
+    }
+
+    @Test
+    public void test_random(){
```

Comment:
All method names must be start with *should_*  for example: should_do_something_when_other_thing....